package.json

```diff
@@ -78,7 +78,7 @@
     "jquery-i18next": "^1.2.1",
     "later": "^1.2.0",
     "libphonenumber-js": "1.4.2",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.12",
```

Comment:
Looks like `4.17.13` is out now as well to fix a few issues that landed in `4.17.12` since we haven't merged yet, let's go ahead and bump to that version.

Reply:
updated.